frontend/src/components/RecordsList.tsx

```diff
@@ -113,6 +113,11 @@ export default function RecordsList({
           [`${dataName}ConnectionWhere2`]: {
             OR: queryFilterWhereVariables(parsedSearchVals),
           },
+          hasCohortSampleSamplesWhere2: {
+            hasMetadataSampleMetadata_SOME: {
+              primaryId_IN: parsedSearchVals,
+            },
+          },
```

Comment:
This needs to be generalized

Reply:
The other tables still render fine without having to generalize this specific "where" variable but for consistency we should probably move forward with generalizing it the way that the other query variables are generalized